<h1><p style="color: green">Conclusion</p></h1>

In [1]:
from prettytable import PrettyTable

In [2]:
ptable2 = PrettyTable()
ptable2.title = " Model Comparision "
ptable2.field_names = ['Model Name','Hyperparameter Tunning', 'categorical_crossentropy', 'Accuracy']

print("\n\n ********************************* Deep Learning LSTM Model Comparision ***********************************")
ptable2.add_row(["LSTM With 1_Layer(neurons:32)","Done","0.410", "0.909"])
ptable2.add_row(["LSTM With 2_Layer(neurons:32, neurons:32)","Done","0.2457", "0.9155"])
ptable2.add_row(["LSTM With 3_Layer(neurons:48)","Done","0.235", "0.918"])
ptable2.add_row(["LSTM With 4_Layer(neurons:64)","Done","0.385", "0.9205"])
ptable2.add_row(["LSTM With 5_Layer(neurons:80)","Done","0.242", "0.9302"])


print(ptable2)



 ********************************* Deep Learning LSTM Model Comparision ***********************************
+----------------------------------------------------------------------------------------------------------+
|                                            Model Comparision                                             |
+-------------------------------------------+------------------------+--------------------------+----------+
|                 Model Name                | Hyperparameter Tunning | categorical_crossentropy | Accuracy |
+-------------------------------------------+------------------------+--------------------------+----------+
|       LSTM With 1_Layer(neurons:32)       |          Done          |          0.410           |  0.909   |
| LSTM With 2_Layer(neurons:32, neurons:32) |          Done          |          0.2457          |  0.9155  |
|       LSTM With 3_Layer(neurons:48)       |          Done          |          0.235           |  0.918   |
|       LSTM With 

2. ___Deep Learning LSTM Model Comparision___ : When we talking about LSTM Model, here with LSTM we are using simple RAW data(in ML model we are using Single engineered data made by an expert), but we can see the result without any FE data, LSTM perform very-very well and got highest 91% accuracy with 2_layer LSTM with hyperparameter Tunning and also we can clearly when we are increasing LSTM layer and Hyperparameter Tunning the cross-entropy value is decreasing and Accuracy is increasing.